dev-docs/bidders/flatads.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: bidder
+title: flatads
+description: Prebid Flatads Bidder Adapter
+biddercode: flatads
+tcfeu_supported: false
+usp_supported: true
```

Comment:
the old US Privacy protocol has been deprecated for more than a year now. You can leave this field, but we recommend adding a `gpp_sids` declaration as described in https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html

```
gpp_sids: tcfeu, tcfca, usnat, usstate_all, usp
```